river/Root.zig

```diff
@@ -74,6 +74,7 @@ pub fn init(self: *Self, server: *Server) !void {
 
     const noop_wlr_output = c.wlr_noop_add_output(server.noop_backend) orelse return error.OutOfMemory;
     try self.noop_output.init(self, noop_wlr_output);
+    c.wlr_output_layout_add_auto(self.wlr_output_layout, noop_wlr_output);
```

Comment:
We shouldn't add the noop output to the layout. It is an implementation detail and should never be advertised.